h2o-r/h2o-DESCRIPTION.template

```diff
@@ -16,7 +38,7 @@ Description: R interface for 'H2O', the scalable open source machine learning
     machine learning algorithm (AutoML).
 License: Apache License (== 2.0)
 URL: https://github.com/h2oai/h2o-3
-BugReports: http://jira.h2o.ai
+BugReports: https://0xdata.atlassian.net/projects/PUBDEV
```

Comment:
Is this the right URL? Did something change?

Reply:
The old URL points to a page that has PUBDEV (h2o-3), Sparkling Water and Steam, so the updated one is the direct link to h2o-3 JIRAs.